### PR TITLE
Added BOATS as a generator tool for swagger/openapi

### PR DESCRIPTION
**Pull request by johndcarmichael**

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -158,6 +158,17 @@
   v2: true
   v3: true
 
+- name: BOATS
+  category: documentation
+  link: https://www.npmjs.com/package/boats
+  github: https://github.com/johndcarmichael/boats
+  language: Node.js
+  description: |
+    BOATS allows for larger teams to contribute to multi-file swagger/openapi files by writing in Nunjucks giving access
```
Comment (matthewtrask) [inline _data/tools.yml]: please remove "swagger" from the description.

File: _data/tools.yml [hunk 2]
```diff
@@ -158,6 +158,17 @@
   v2: true
   v3: true
 
+- name: BOATS
+  category: documentation
```
Comment (philsturgeon) [inline _data/tools.yml]: Is this documentation? I know a lot of folks use OpenAPI for documentation, but this tool is helping folks write OpenAPI so if anything its almost like [a DSL](https://developer.squareup.com/blog/making-openapi-swagger-bearable-with-your-own-dsl/). Should we start a DSL category?

Comment (matthewtrask) [reply _data/tools.yml]: Might be useful to have a DSL category.

Comment (johndcarmichael) [reply _data/tools.yml]: Cool, I update the PR. I would agree that boats is a kind of DSL, all be it a lighter version of the one you posted @philsturgeon

**Pull request merged**